src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -489,18 +489,29 @@ def encode_prompt(
 
         return prompt_embeds, negative_prompt_embeds
 
-    def encode_image(self, image, device, num_images_per_prompt):
+    def encode_image(self, image, device, num_images_per_prompt, output_hidden_states):
```

Comment:
```suggestion
    def encode_image(self, image, device, num_images_per_prompt, output_hidden_states=None):
```

This is public API so we need to make sure it's backward compatible. Also ideally we should add docstrings here